boot/freeldr/freeldr/ntldr/wlmemory.c

```diff
@@ -295,6 +295,9 @@ WinLdrSetupMemoryLayout(IN OUT PLOADER_PARAMETER_BLOCK LoaderBlock)
         }
     }
 
+    
+    MempSetupPaging((PFN_NUMBER)AcpiTableSource / PAGE_SIZE, Extension->AcpiTableSize / PAGE_SIZE, FALSE);
```

Comment:
This isn't handled by the loops above (the one at lines 249-261)?

Reply:
No?

Reply:
No wait i was too tired when i was doing this, you're right.